token/token.go

```diff
@@ -31,11 +32,12 @@ func NewManager(publicKey *rsa.PublicKey, privateKey *rsa.PrivateKey) Manager {
 	}
 }
 
+// Generate generates a new JWT token. For tests only.
 func (mgm tokenManager) Generate(ident account.Identity) (string, error) {
 	token := jwt.New(jwt.SigningMethodRS256)
 	token.Claims.(jwt.MapClaims)["uuid"] = ident.ID.String()
-	token.Claims.(jwt.MapClaims)["fullName"] = ident.FullName
-	token.Claims.(jwt.MapClaims)["imageURL"] = ident.ImageURL
+	token.Claims.(jwt.MapClaims)["username"] = ident.Username
```

Comment:
I don't think so. I was planning to fix the token generation in further commits when fixing tests.